prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html

```diff
@@ -73,75 +131,186 @@ <h2>BC Services Card</h2>
+
+          <ul>
+            <li>
+              PharmaNet Administrator
+
+              <app-contextual-help>
+                Either directly sets up individual PharmaNet access accounts at a site or
+                has the authority to request accounts from the site's contracted PharmaNet software vendor. They may,
+                for example, be the site’s office administrator, HR lead, medical office assistant (MOA), or
+                practitioner. This may be the software vendor.
+              </app-contextual-help>
+            </li>
+            <li>
+              Privacy Officer
 
-      <app-pill>
-        <section>
-          <h2>Get Support</h2>
+              <app-contextual-help>
+                Under privacy legislation (PIPA and FOIPPA), organizations holding or accessing
+                personal information must designate a Privacy Officer, who responds to privacy and information
+                management concerns on the organization’s behalf. In the event of a security breach, or concerns with
+                how PharmaNet data is accessed or used at the site, the Privacy Officer will be asked to help with
+                investigations and provide critical information quickly.
+              </app-contextual-help>
+            </li>
+            <li>
+              Technical Support Contact
 
+              <app-contextual-help>
+                This individual may be either a staff person at the site, a corporate IT
+                resource, or an independent IT contractor. They will be contacted in the event of a technology-related
+                issue (e.g., malware, problematic network traffic). They should familiar with: the practice's network
+                structure, Internet connections in use at the site, the application(s) used to access PharmaNet at the
+                site and security mechanisms such as firewalls and anti-virus measures (generally and at the site).
+              </app-contextual-help>
+            </li>
+          </ul>
+
+          <h2>User Guides</h2>
           <p>
-            Questions about PRIME should be directed to:
-            <br>
-            <app-prime-phone></app-prime-phone>
-            <br>
-            <app-prime-support-email></app-prime-support-email>
+            Click here for your guides.
+            <a href="https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/pharmacare/pharmanet-bc-s-drug-information-network/prime/prime-user-guides">PRIME user guides</a>
           </p>
         </section>
-      </app-pill>
+      </div>
+    </div>
 
+    <div class="col-sm-12 col-md-4"
+         *ngIf="mode === 'health-authority'">
+      <div>
+        <div class="graphic-wrapper">
+          <img class="graphic"
+               src="/assets/enrolment_v2.svg"
+               alt="Notification Icon">
+        </div>
+        <section class="content">
+          <h2>User Guides</h2>
+          <p>
+            Click here for your guides.
```

Comment:
I think that should be "Click here for **user** guides**:**"

Reply:
it is fixed.